data/img/material/black/move_down.svg

```diff
@@ -0,0 +1,4 @@
+<svg version="1.1" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+ <path d="M0 0h24v24H0V0z" fill="none"/>
```

Comment:
I think you can remove this line from all SVGs without changing anything - no fill, no stroke, no difference. It's just an invisible box the size of the image.

Reply:
I removed it and it seems like it doesn't change anything visually. Thanks :)